python/config_settings/BUILD.bazel

```diff
@@ -170,3 +170,13 @@ string_flag(
 define_pypi_internal_flags(
     name = "define_pypi_internal_flags",
 )
+
+string_flag(
+    name = "free_threading",
+    build_setting_default = "no",
+    values = [
+        "yes",
+        "no",
+    ],
```

Comment:
See `UniversalWhlFlag` symbol and please use the enum class for the `string_flag` wiring.